--- Parse/Binary/FixedFormat.pm.bak	Fri Dec 24 22:01:18 2004
+++ Parse/Binary/FixedFormat.pm	Thu Jan 22 15:37:36 2009
@@ -140,7 +140,13 @@
 	if ($self->{Count}[$i]) {
 	    push @flds,map {ref($_) ? @$_ : $_} @{$rec->{$name}};
 	} else {
-	    push @flds,$rec->{$name};
+	    if (ref($rec->{$name}) eq "ARRAY") {
+                if ($#{$rec->{$name}} >= 0) {
+                    push @flds,$rec->{$name};
+                }
+            } else {
+                push @flds,$rec->{$name};
+            }
 	}
     	$i++;
     } 
